src/com/esotericsoftware/kryo/io/ByteBufferInput.java

```diff
@@ -352,6 +352,16 @@ public void readBytes (byte[] bytes, int offset, int count) throws KryoException
 		}
 	}
 
+	public long readBytesAsLong (int count) {

```

Comment:
This is better than my initial idea of `readBytes2`, `readBytes3`, etc. Nice!

I don't hate the name `readBytesAsLong`, but what if it was `readLong(int count)`? Should we have a `readInt(int count)` for 2-3 bytes?

The first line of the method should be:
```
if (count < 0 || count > 8) throw new IllegalArgumentException("count must >= 0 and <= 8: " + count);
```
Same with other methods taking a `count` parameter.

Reply:
Done in 029190b, and the `readInt(int count)` in 9698899.